db/migrate/20160301070319_add_version_ostree_branches.rb

```diff
@@ -0,0 +1,30 @@
+class AddVersionOstreeBranches < ActiveRecord::Migration
+  def up
+    drop_table :katello_ostree_branches if ActiveRecord::Base.connection.table_exists? "katello_ostree_branches"
+
+    create_table :katello_ostree_branches do |t|
```

Comment:
It is better to drop the table entirely and recreate it instead of migrating the current table?